WatchApp/WatchApp Watch App/Pages/ChatView.swift

```diff
@@ -1,112 +1,219 @@
                 }
-            }.listStyle(PlainListStyle()).padding(.bottom, 10)
-            
-            
+                .frame(maxHeight: .infinity).padding(.bottom, 10)
             HStack() {
-                TextField("Message...", text: $newMessage).frame(width: 120.0, height: 0)
+                Button(action: {}) {
+                    NavigationLink(destination: SettingView()){
+                        Image(systemName: "gearshape.fill")
+                            .font(.system(size: 20))
+                            .frame(maxWidth: .infinity)
+                    }
+                }.frame(width: 45, height: 45)
+                    .cornerRadius(5).buttonStyle(PlainButtonStyle())
+                
+                TextField("", text: $newMessage, onEditingChanged: { (isChanged) in
+                    if isStarted && !isChanged {
+                        isStarted = false
+                        print(newMessage)
+                    }
+                }).frame(height: 45)
+                    .overlay(
+                        RoundedRectangle(cornerRadius: 6)
+                            .stroke(Color.white, lineWidth: 1)
+                    )
                 Button(action: {
-                    let keychain = KeychainSwift()
+                    
+                    sendNotification()
+                }) {
+                    Image(systemName: "paperplane.fill")
+                        .font(.system(size: 20))
+                        .frame(maxWidth: .infinity)
+                }.frame(width: 45, height: 45)
+                    .cornerRadius(5)
+            }
+        }.edgesIgnoringSafeArea(.bottom)
+    }
+
+    func sendNotification() {
+        if newMessage == "" {
+            return
+        }
+        // Try to get the UUID from the keychain
+        let keychain = KeychainSwift()
 
-                    // Try to get the UUID from the keychain
-                    if keychain.get("uuid") == "" {
-                        let uuid = "956a11be45cba4a4"
-                        keychain.set(uuid, forKey: "uuid")
+        if keychain.get("uuid") == "" {
+            let uuid = "956a11be45cba4a4"
+            keychain.set(uuid, forKey: "uuid")
+        }
+        
+        // Add sent message to chat list
+        let message = ChatMessage(
+            type: TypeChat.SEND,
+            message: newMessage,
+            widgetType: TypeWidget.EMPTY,
+            widgetDesc: ""
+        )
+        messages.append(message)
+
+        if let url = URL(string: Constants.URL_SEND_NOTIFICATION) {
+            var request = URLRequest(url: url)
+            request.httpMethod = "POST"
+            request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+
+            let postData: [String: Any] = ["confs": Constants.getConfs(uuid: uuid), "message": newMessage]
+
+            do {
+                let jsonPostData = try JSONSerialization.data(withJSONObject: postData, options: .prettyPrinted)
```

Comment:
Please map it automatically as an object including result, message, status_code.